### PR TITLE
Adding distribution metrics FAQ

### PR DESCRIPTION
**Pull request by apigirl**
### What does this PR do?
Adds a FAq for updating to Distribution metrics

https://docs-staging.datadoghq.com/kaylyn/metrics-guide/metrics/faq/updating-to-distribution-metrics-faq/

### CODE REVIEW

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+is_beta: true
+further_reading:
+- link: "/logs/faq/how-to-send-logs-to-datadog-via-external-log-shippers/"
+  tag: "FAQ"
+  text: "How to Send Logs to Datadog via External Log Shippers?"
+---
+
+## Why has my Distribution Metrics page been deprecated?
+
+Datadog has a new pipeline for calculating globally accurate percentiles for your distribution metrics - this unlocks more query functionality and an easier configuration workflow for distributions.
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:13]: Datadog has released a new pipeline for calculating globally accurate percentiles for your distribution metrics - this unlocks more query functionality and an easier configuration workflow for distributions.

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+## How does this new pipeline for Distributions affect me? 
+
+- Unlocks new query functionalities:
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:17]: **Unlocks new query functionalities:**

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  - Accurate percentiles in time for any historical timeframe (i.e., the past day, week, month) of their choosing.
+  - Boolean filtered queries are now available with distribution metrics with percentiles, so that you can write queries like: `p99:request_latency_distribution{app:A OR app:B} by {app}`.
+- Easier tag configuration and percentile aggregation workflows:
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:20]: - **Easier tag configuration and percentile aggregation workflows:**

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+---
+
+## Why has my Distribution Metrics page been deprecated?
+
+Datadog has a new pipeline for calculating globally accurate percentiles for your distribution metrics - this unlocks more query functionality and an easier configuration workflow for distributions.
+
+## How does this new pipeline for Distributions affect me? 
+
+- Unlocks new query functionalities:
+  - Accurate percentiles in time for any historical timeframe (i.e., the past day, week, month) of their choosing.
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:18]: - Accurate percentiles in time for any historical timeframe (i.e., the past day, week, month)

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+  - Boolean filtered queries are now available with distribution metrics with percentiles, so that you can write queries like: `p99:request_latency_distribution{app:A OR app:B} by {app}`.
+- Easier tag configuration and percentile aggregation workflows:
+  - The Metrics Summaray page now centrally manages all tag configuration and percentile aggregations.
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:21]: - All tag configuration / percentile aggregations will be managed in one centralized location on the [Metrics Summary](https://app.datadoghq.com/metric/summary) page

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+  - Use the the “Manage Tags” button on the Metrics Summary details side panel to manage all tag configurations for your distribution metrics.
+  - To add percentile aggregations to your distribution, navigate to the new percentiles section on the side panel, click *Edit* and select “Include percentile aggregations” for the same queryable tags defined as your baseline distribution. You’ll no longer need to specify an additional configuration rule.
+- More Intuitive number of Custom Metrics from Distributions with Percentiles:
+  The number of custom metrics emitted from additional percentile aggregations is no longer 5 percentile timeseries for every potentially queryable tag value combination. The new number of custom metrics emitted from distributions with percentiles is now counted more similarly to our other metric types:
+
+  | Metric Type                               | Query Type                                                                                                     | # of billable custom metrics emitted               |
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:27]: | Metric Type                               | Query Type                                                                                                     | # of custom metrics emitted               |

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+|-------------------------------------------|----------------------------------------------------------------------------------------------------------------|----------------------------------------------------|
+| Distribution                              | Queryable by `count`, `sum`, `min`, `max`, and `avg`.                                                          | 5 * (tag value combinations emitted in your data)  |
+| Distribution with Percentile Aggregations | Queryable by `count`, `sum`, `min`, `max`,`avg` + p50, `avg` + p75, `avg` + p90, `avg` + p95, and `avg` + p99. | 10 * (tag value combinations emitted in your data) |
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:30]: | Distribution with Percentile Aggregations | Queryable by `count`, `sum`, `min`, `max`,`avg`, `p50`, `p75`, `p90`, `p95`, and `p99`. | 10 * (tag value combinations emitted in your data) |

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+{{< img src="metrics/faq/sketch-db-migration-faq/configure-tags.png" alt="Configure tags">}}
+
+Clicking the *Manage Tags* button allows you to configure which tags are queryable on a particular metric. You’ll be able to preview the new number of indexed custom metrics from your tag configuration before saving it.
+
+{{< img src="metrics/faq/sketch-db-migration-faq/manage-tags.png" alt="Manage tags">}}
+
+## How can I add percentiles to my Distribution metric? 
+
+Click *edit* and “Include percentile aggregations” on the same queryable tags defined as your baseline distribution.
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:43]: Click *Edit* and “Include” to include percentile aggregations on the same queryable tags defined as your baseline distribution.

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 9]
```diff
@@ -0,0 +1,55 @@
...
+
+{{< img src="metrics/faq/sketch-db-migration-faq/add-percentiles.gif" alt="Add percentiles to a distribution metric">}}
+
+## How can I add percentiles to multiple Distribution metrics?
+
+Click the *Calculate Percentiles* button on the Metrics Summary page and use * for wildcard metric name(s).
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:49]: Click the *Calculate Percentiles* button on the Metrics Summary page and use * for wildcarding metric names.

**Review by ijkaylin — CHANGES_REQUESTED** [review]

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+{{< img src="metrics/faq/sketch-db-migration-faq/bulk-percentiles.gif" alt="Add percentiles to multiple distribution metrics">}}
+
+
+
```
Comment (apigirl) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:54]:
```suggestion
[1]: https://app.datadoghq.com/metric/summary
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+  text: "How to Send Logs to Datadog via External Log Shippers?"
+---
+
+## Why has my Distribution Metrics page been deprecated?
+
+Datadog has released a new pipeline for calculating globally accurate percentiles for your distribution metrics - this unlocks more query functionality and an easier configuration workflow for distributions.
+
+## How does this new pipeline for Distributions affect me? 
+
+**Unlocks new query functionalities:**```
```
Comment (apigirl) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:17]:
```suggestion
- **Unlocks new query functionalities:**
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+---
+
+## Why has my Distribution Metrics page been deprecated?
+
+Datadog has released a new pipeline for calculating globally accurate percentiles for your distribution metrics - this unlocks more query functionality and an easier configuration workflow for distributions.
+
+## How does this new pipeline for Distributions affect me? 
+
+- **Unlocks new query functionalities:**
+  - Accurate percentiles in time for any historical timeframe (i.e., the past day, week, month).
```
Comment (kayayarai) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:18]:
```suggestion
  - Accurate percentiles in time for any historical timeframe (such as the past day, week, or month).
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+is_beta: true
+further_reading:
+- link: "/logs/faq/how-to-send-logs-to-datadog-via-external-log-shippers/"
+  tag: "FAQ"
+  text: "How to Send Logs to Datadog via External Log Shippers?"
+---
+
+## Why has my Distribution Metrics page been deprecated?
+
+Datadog has released a new pipeline for calculating globally accurate percentiles for your distribution metrics - this unlocks more query functionality and an easier configuration workflow for distributions.
```
Comment (kayayarai) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:13]:
```suggestion
Datadog has released a new pipeline for calculating globally accurate percentiles for your distribution metrics --- this unlocks more query functionality and an easier configuration workflow for distributions.
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+
+## How does this new pipeline for Distributions affect me? 
+
+- **Unlocks new query functionalities:**
+  - Accurate percentiles in time for any historical timeframe (i.e., the past day, week, month).
+  - Boolean filtered queries are now available with distribution metrics with percentiles, so that you can write queries like: `p99:request_latency_distribution{app:A OR app:B} by {app}`.
+- **Easier tag configuration and percentile aggregation workflows**:
+  - All tag configuration / percentile aggregations will be managed in one centralized location on the [Metrics Summary][1] page
```
Comment (kayayarai) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:21]:
```suggestion
  - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+  - Use the the “Manage Tags” button on the Metrics Summary details side panel to manage all tag configurations for your distribution metrics.
+  - To add percentile aggregations to your distribution, navigate to the new percentiles section on the side panel, click *Edit* and select “Include percentile aggregations” for the same queryable tags defined as your baseline distribution. You’ll no longer need to specify an additional configuration rule.
+- More Intuitive number of Custom Metrics from Distributions with Percentiles:
```
Comment (kayayarai) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:24]:
```suggestion
- **More usable number of custom metrics from distributions with percentiles**:
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 16]
```diff
@@ -0,0 +1,54 @@
...
+  The number of custom metrics emitted from additional percentile aggregations is no longer 5 percentile timeseries for every potentially queryable tag value combination. The new number of custom metrics emitted from distributions with percentiles is now counted more similarly to our other metric types:
+
+  | Metric Type                               | Query Type                                                                                                     | # of custom metrics emitted               |
+|-------------------------------------------|----------------------------------------------------------------------------------------------------------------|----------------------------------------------------|
+| Distribution                              | Queryable by `count`, `sum`, `min`, `max`, and `avg`.                                                          | 5 * (tag value combinations emitted in your data)  |
+| Distribution with Percentile Aggregations | Queryable by `count`, `sum`, `min`, `max`, p50,  p75,  p90, p95, and p99. | 10 * (tag value combinations emitted in your data) |
```
Comment (kayayarai) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:30]:
```suggestion
| Distribution with percentile aggregations | Queryable by `count`, `sum`, `min`, `max`, `p50`,  `p75`,  `p90`, `p95`, and `p99`. | 10 for each tag-value combination emitted in your data |
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 17]
```diff
@@ -0,0 +1,54 @@
...
+- More Intuitive number of Custom Metrics from Distributions with Percentiles:
+  The number of custom metrics emitted from additional percentile aggregations is no longer 5 percentile timeseries for every potentially queryable tag value combination. The new number of custom metrics emitted from distributions with percentiles is now counted more similarly to our other metric types:
+
+  | Metric Type                               | Query Type                                                                                                     | # of custom metrics emitted               |
+|-------------------------------------------|----------------------------------------------------------------------------------------------------------------|----------------------------------------------------|
+| Distribution                              | Queryable by `count`, `sum`, `min`, `max`, and `avg`.                                                          | 5 * (tag value combinations emitted in your data)  |
```
Comment (kayayarai) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:29]:
```suggestion
| Distribution                              | Queryable by `count`, `sum`, `min`, `max`, and `avg`.                                                          | 5 for each tag-value combination emitted in your data  |
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+| Distribution with Percentile Aggregations | Queryable by `count`, `sum`, `min`, `max`, p50,  p75,  p90, p95, and p99. | 10 * (tag value combinations emitted in your data) |
+
+
+## How can I configure tags on my Distribution metric? 
+
+{{< img src="metrics/faq/sketch-db-migration-faq/configure-tags.png" alt="Configure tags">}}
+
+Clicking the *Manage Tags* button allows you to configure which tags are queryable on a particular metric. You’ll be able to preview the new number of indexed custom metrics from your tag configuration before saving it.
```
Comment (kayayarai) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:37]:
```suggestion
Click the **Manage Tags** button to configure which tags are queryable on a particular metric. You can preview the new number of indexed custom metrics from your tag configuration before saving it.
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 19]
```diff
@@ -0,0 +1,54 @@
...
+
+{{< img src="metrics/faq/sketch-db-migration-faq/manage-tags.png" alt="Manage tags">}}
+
+## How can I add percentiles to my Distribution metric? 
+
+Click *Edit* and “Include” to include percentile aggregations on the same queryable tags defined as your baseline distribution.
```
Comment (kayayarai) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:43]:
```suggestion
Click **Edit** and select **Include** to include percentile aggregations on the same queryable tags defined as your baseline distribution.
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+
+{{< img src="metrics/faq/sketch-db-migration-faq/add-percentiles.gif" alt="Add percentiles to a distribution metric">}}
+
+## How can I add percentiles to multiple Distribution metrics?
+
+Click the *Calculate Percentiles* button on the Metrics Summary page and use * for wildcarding metric names.
```
Comment (kayayarai) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:49]:
```suggestion
Click **Calculate Percentiles** on the Metrics Summary page and use `*` for wildcarding metric names.
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
A few edits

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 21]
```diff
@@ -0,0 +1,54 @@
+---
+title: Updating to Distribution Metrics FAQ
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:2]:
```suggestion
title: Update to Distribution Metrics Workflow FAQ
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 22]
```diff
@@ -0,0 +1,54 @@
...
+Datadog has released a new pipeline for calculating globally accurate percentiles for your distribution metrics --- this unlocks more query functionality and an easier configuration workflow for distributions.
+
+## How does this new pipeline for Distributions affect me? 
+
+- **Unlocks new query functionalities:**
+  - Accurate percentiles in time for any historical timeframe (such as the past day, week, or month).
+  - Boolean filtered queries are now available with distribution metrics with percentiles, so that you can write queries like: `p99:request_latency_distribution{app:A OR app:B} by {app}`.
+- **Easier tag configuration and percentile aggregation workflows**:
+  - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page
+  - Use the the “Manage Tags” button on the Metrics Summary details side panel to manage all tag configurations for your distribution metrics.
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:22]:
```suggestion
  - Use the “Manage Tags” button on the Metrics Summary details side panel to manage the queryable tags configuration for a distribution metric.
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 23]
```diff
@@ -0,0 +1,54 @@
...
+
+Datadog has released a new pipeline for calculating globally accurate percentiles for your distribution metrics --- this unlocks more query functionality and an easier configuration workflow for distributions.
+
+## How does this new pipeline for Distributions affect me? 
+
+- **Unlocks new query functionalities:**
+  - Accurate percentiles in time for any historical timeframe (such as the past day, week, or month).
+  - Boolean filtered queries are now available with distribution metrics with percentiles, so that you can write queries like: `p99:request_latency_distribution{app:A OR app:B} by {app}`.
+- **Easier tag configuration and percentile aggregation workflows**:
+  - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:21]:
```suggestion
  - All tag configuration-percentile aggregations are managed in one centralized location on the [Metrics Summary][1] page.
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 24]
```diff
@@ -0,0 +1,54 @@
...
+  - Use the the “Manage Tags” button on the Metrics Summary details side panel to manage all tag configurations for your distribution metrics.
+  - To add percentile aggregations to your distribution, navigate to the new percentiles section on the side panel, click *Edit* and select “Include percentile aggregations” for the same queryable tags defined as your baseline distribution. You’ll no longer need to specify an additional configuration rule.
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:23]:
```suggestion
  - To add percentile aggregations to your distribution, navigate to the new percentiles section on the side panel, click *Edit* and select “Include” for percentile aggregations on the same queryable tags defined on your baseline distribution. You’ll no longer need to specify an additional configuration rule.
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 25]
```diff
@@ -0,0 +1,54 @@
...
+- **More usable number of custom metrics from distributions with percentiles**:
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:24]:
```suggestion
- **More intuitive number of custom metrics from distributions with percentiles**:
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 26]
```diff
@@ -0,0 +1,54 @@
...
+  The number of custom metrics emitted from additional percentile aggregations is no longer 5 percentile timeseries for every potentially queryable tag value combination. The new number of custom metrics emitted from distributions with percentiles is now counted more similarly to our other metric types:
+
+  | Metric Type                               | Query Type                                                                                                     | # of custom metrics emitted               |
+|-------------------------------------------|----------------------------------------------------------------------------------------------------------------|----------------------------------------------------|
+| Distribution                              | Queryable by `count`, `sum`, `min`, `max`, and `avg`.                                                          | 5 for each tag-value combination emitted in your data  |
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:29]:
```suggestion
| Distribution                              | Queryable by `count`, `sum`, `min`, `max`, and `avg`.                                                          | 5 for each tag value combination emitted in your data  |
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 27]
```diff
@@ -0,0 +1,54 @@
...
+| Distribution with percentile aggregations | Queryable by `count`, `sum`, `min`, `max`, `p50`,  `p75`,  `p90`, `p95`, and `p99`. | 10 for each tag-value combination emitted in your data |
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:30]:
```suggestion
| Distribution with percentile aggregations | Queryable by `count`, `sum`, `min`, `max`, `p50`,  `p75`,  `p90`, `p95`, and `p99`. | 10 for each tag value combination emitted in your data |
```

File: content/en/metrics/faq/updating-to-distribution-metrics-faq.md [hunk 28]
```diff
@@ -0,0 +1,54 @@
...
+
+{{< img src="metrics/faq/sketch-db-migration-faq/configure-tags.png" alt="Configure tags">}}
+
+Click the **Manage Tags** button to configure which tags are queryable on a particular metric. You can preview the new number of indexed custom metrics from your tag configuration before saving it.
+
+{{< img src="metrics/faq/sketch-db-migration-faq/manage-tags.png" alt="Manage tags">}}
+
+## How can I add percentiles to my Distribution metric? 
+
+Click **Edit** and select **Include** to include percentile aggregations on the same queryable tags defined as your baseline distribution.
```
Comment (ijkaylin) [inline content/en/metrics/faq/updating-to-distribution-metrics-faq.md:43]:
```suggestion
Click **Edit** and select **Include** to include percentile aggregations on the same queryable tags defined on your baseline distribution.
```

**Review by ijkaylin — CHANGES_REQUESTED** [review]

**Review by kayayarai — APPROVED** [review]

**Pull request merged**